src/components/TopNav/index.tsx

```diff
@@ -0,0 +1,112 @@
sK2ZG1nXmblF09c0wOaCyVvn1fD55/7u2fmMI+dvsejIxb/M6dL2aDtQ92z+/r07OEka+Uc4ARrazgxRUcHe6xToAE8l0cBg07DhMelHgWgk8ARHzvgJMC32wDfLaN9x7YjTfun4X/h3cXHnvjy5dPNd+2OE2G1H6QBCgsODBGwYUCVvRuTiULVb1OalvhqGvSunrgo4ZlonYhtEt5lZH6HYKpf4YojuRdqwaKsLZnKd9Qrry7d6j+5hOu/8z9muaxt+73BkbSDb9j65GV3u7v1hb2H1KuJIXAUggmUYVGVOEtgNBrW/EdCHQApELAKAonKwBMRHGOvmG+bMO4FIQnxzfVG9g9m+FV13+HfRn73R/NfeG4G49bOt38zvImVnaZheljxqMFJDUmBlOgPEOuRw3FNqQ8CyHXPOq9YnnGBMdubE9gJ7sTDQcB0BUzOs+CqB3JvtsSPSobebZjS3fXK5545/t+r/vy/V7BSLol75hc2C43LqgO9f6jUNu8QTsLK7e9R9UgChyxalRUyYG9AJ45ABEFDIkhilvBNzRNdEMBy6Jsmuw7bamWNXF12sS513+f7VO9e797YO6zR/968YI2e9MBs/inIWPosewgmFsWVeyQfYSAYEcAIR5IGaltIO+FBvXInxygZcFzTtlODKYUfAj4gVOVzr8vd0aZEIbqe8qlX4+wyl+dffe7f+9y+nsHI+04W/jO3X+R9FX+u2uob6BUCdU2V/nVqwiBJ1KvciDSQTXLPQPy9iNeDHB0Qm6hitUJkLIChkTek+OHGrNoC9XsvWc/Am+64AK6t8p+91Bw/3P0tc9fONv+wJIWW1AzAhl34btWi4fjhQLWw4ntxYIEuReM5+HZ3KodCwGLAlvIokKG5sc6hcDVSXW0v20hqNuQzd5XZh+8amXvmy+4/II2/gDuDwRG2g2/bWRFqZq9pTLQ/8Jaf5VZW1IROAQAAYQgkhWDUuxPAEj0uzkAjjDXnCpW+G7kd6GaJ85bTWUfulrYh176m2+xm7HfPWTdp9fevnTh1Oh7lzTYXw8L47be/yxiJcwPEoy73DWbMsKTm51f1FXPO6pfzk7FQsCIgYe58AEQVgUUdeEXq5Vyb7VxoZZtSHDVVKX7H566/oLb8Qd0f1Awsm7xuyaewav8vbWBnsMq3WXHiIpAQf0CIQB1UsEiwHDvmveTPfjT0a8sDz6F7Cr+Ti5TItqU2VlsbGd4V2k9Pnv99ez3Yvzb7/a9+/IRVzx1fhMfXNLCof1yvSHHlEwlZhQIiit9eJ/acLJIpeOB39S+0xlAInCbI+zibyyDyjAtBHUz45t3l0tv25qVv/jCey+YxR/Y/VHASDnO2YK3jb6k3Fd5XXWg6+ByrZwzEM8FPJ3YyVysBXOBGFETc2wrK2ZGc/klQUioZKPNJr7QrOMtd36XTWK/+5Nz54Mnqw/75UsXNvnrF6XJgb1geg88dFJ5bGUvAiPKfijYcBSDR4biHrgQsFDgb+wXAsCCui83qqxLlYxl4ztKycd4ib1HsKEJ/JHcHw+MjFt6/sSCtJa+IOmqvKY6r7ZUglJiBmntLRMqUuXmYjh79Hsvvy9iVKngPa06ZoRa9pVZhvesu5Ddjf3uT959/vhfLOyeSP9uKMVrBCgt7k0Ssz15sfoTg1HxO+Y3mFVP/HTTPch3nWxUMUgZNdEDE1QPmv1mRrSW21k2u6uMT0+Vuj7ynHVv+KPL6R8djJz7NK8s3Db2glJ3+eXl/urxle4KSxLmKnziyzIEHgpE0f1CkCliT5Y5A+HGi7zYvzhcWcbtplwtEhsbdQFCXXjvvV9kY9jv/uzcFw66rKtcxd8ONtOXz0+T4weE+lZRo/IJSFA7EQtBKGAtdFCiu44M5lYAeWzULmZfebDj0IbsFKnwY4KnfAfDptEK+2y91v3xc29/9W48RNxDB4ysO58ni/j4E5Ku0otKtfKzBTBV1Rgl1sGgXARQnVjQHGwonicX251ihsSNPSgVvWNCFbsWs/hEcwzfXX8J+6PR3P3uD+fOf+xl5YPvazy2hycvFWzpWcM8KfcKIbXrXdGJtW5cDyKjNfO9bDm7jmNJkboWzdS3YBV8Z4WW60m9s0JYR1iajSbsiukyPjaJ2R/97R9RHevkHnpgRNzQ6/lAtTbxAlYtPSvprjxGGLvL5aqmx4GalO1ZzQpAJsu/U2SEdkAFDT5WDUvragrHrVkD30ia+NJvv75/UuvD2X3imFuGeupbnzvQTs/tz3DakACmHtFdUjYjumNjceH0ERQZtLMck4qZVu5cgFUqbUHifIy1+Tjj109V8EPe1fXlZ9/2yrvwEHYPaTCibun5MyvTtHFCqVJ7Fquwx5SqpZXlWomV5BKxRJ3LGcHnYEh28nUMRiDsh8vtihpCBWtgjLfw67TJvi1sfpfd82Wsg+/r2+/2O+U+svqy5QOYelRvljyzO+WP6+d8WR9PWDfkZg7cbeSQU7GEsAUjuym7ygEUEVLx2xbfNtDGlDAUTSAdrZeSKwUYfXe2zH/21+tedQ/7E5HTPxkwit2K1/FlrdLsk8Cax5eqtaOSUnI8K/GhUkkwJ9EkKYBCvG94nlFZ0FHd8C1Bp+XRZk1Rttemrey2rM1uFRarX95zAG7D/kGK+939dP9z/BXLmtPjTy03G8f0gx1V5fy4asbn9wgBrAloqgBqa0a5fEawZqMZo8SNoOpFZDMhlgJ4JPMRz2dLaLQ5v67O0ttmS0JWq12X3Xnu5jsuuOCCP0k5/ZMFo5w7h5eWrcAyVOorkLbnJ0n5QNEOHSJ400KW8H5kSU00HGrJlCQTCjRns4yn08hKuwX72ShgaxPPSvcJe+RmNh9b772A/cHHWex3f/7ufJyfLD/22GWl6WxlmWXzWcoPrCSlg0o8XSLksy9BWhNgJPi+2kakzRlvZHJ1mQS7kyzb0Ez45naJbSqV+MbdrZltr1z/yj+bPfT+P/eEvEiGei7MAAAAAElFTkSuQmCC';
```

Comment:
can we move this to: `TopNav/logo.ts` and do an `export` from that file, cleans up the JS.

Reply:
I did it